### PR TITLE
Update changelog for development post 1.8

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Updates the changelog template to merge new PRs for the 1.8.x and 1.9 milestones.

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @borda @rohitgr7 @Felonious-Spellfire

### CODE REVIEW

**Review by rohitgr7 — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

File: src/lightning_app/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,55 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [unreleased] - 2023-XX-XX
```
Comment (Borda) [inline src/lightning_app/CHANGELOG.md:8]:
```suggestion
## [unreleased] - 202Y-MM-DD
```

File: src/pytorch_lightning/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,54 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [unreleased] - 2023-XX-XX
```
Comment (Borda) [inline src/pytorch_lightning/CHANGELOG.md:8]:
```suggestion
## [unreleased] - 202Y-MM-DD
```

**Comment by Borda** [comment]
Shall we include/create chlog for Lite?

**Comment by awaelchli** [comment]
@Borda Added it for lite ✅

**Review by akihironitta — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Pull request merged**